src/io/flutter/module/FlutterModuleBuilder.java

```diff
@@ -89,22 +156,40 @@ public ModuleType getModuleType() {
 
   private static void createProjectFiles(@NotNull final ModifiableRootModel model,
                                          @NotNull final VirtualFile baseDir,
-                                         @NotNull final FlutterSdk sdk) {
+                                         @NotNull final FlutterSdk sdk) throws ConfigurationException {
+
+    runFlutterCreate(sdk, model.getModule(), baseDir);
+
     try {
       DartPlugin.ensureDartSdkConfigured(model.getProject(), sdk.getDartSdkPath());
       FlutterSdkUtil.updateKnownSdkPaths(sdk.getHomePath());
     }
     catch (ExecutionException e) {
       LOG.warn("Error configuring the Flutter SDK.", e);
+      throw new ConfigurationException("Error configuring Flutter SDK");
     }
+  }
 
+  private static void runFlutterCreate(@NotNull FlutterSdk sdk, Module module, @NotNull VirtualFile baseDir)
+    throws ConfigurationException {
     // Create files.
     try {
-      sdk.run(FlutterSdk.Command.CREATE, model.getModule(), baseDir, null, baseDir.getPath());
+      final Process process =
+        sdk.startProcess(FlutterSdk.Command.CREATE, module, baseDir, null, baseDir.getPath());
+      if (process != null) {
+        // Wait for process to finish. (We overwrite some files, so make sure we lose the race.)
+        // TODO(skybrian) the user sees a short pause. Show progress somehow?
```

Comment:
For IDEA this runs after the project wizard disappears and before the new project window appears - not ideal.

For WebStorm, the new project window is open, but blank.